nmosauth/auth_server/security_api.py

```diff
@@ -319,16 +319,17 @@ def delete_user(self, username):
     # route for JSON Web Key
     @route(AUTH_VERSION_ROOT + JWK_ENDPOINT + '/', methods=['GET'], auto_json=True)
     def get_jwk(self):
+        current_time = int(time())  # Current UTC Time
+        kid = 'x-nmos-{}'.format(current_time)
         try:
             with open(PUBKEY_PATH, 'r') as myfile:
                 pub_key = myfile.read()
             obj = jwk.dumps(
-                pub_key, kty='RSA', use="sig", key_ops="verify",
-                alg="RS512", kid=date.today().strftime("%d-%m-%y")
+                pub_key, kty='RSA', use="sig", key_ops="verify", alg="RS512", kid=kid
             )
             return (200, obj)
```

Comment:
It looks like this may still need to be wrapped in the following format to ensure it's a JWK set:

```
{
  "keys": [
    {"kid": ...}
  ]
}
```

Reply:
Are we mandating that all JSON Web Keys only be presented as part of a JSON Web Key Set? I assumed for a single key it may be ok to just present the JWK on its own

Reply:
I think we'll always use a set, otherwise clients need to have extra implementation to support both possibilities. Commercial auth servers will only expose sets as far as I'm aware.